test/api/todos.coffee

```diff
@@ -184,3 +251,34 @@ describe "Todos", ->
           expect(body.err).to.equal "Task not found."
           done()
 
+describe "habits", ->
+
+  it "Creates a completed a habit when using up url", (done) ->
+    upHabit = undefined
+    request.post(baseURL + "/user/tasks/withUp/up").send(
```

Comment:
I don't think you're actually creating a habit here. You're actually scoring the todo you created earlier in the test.